### PR TITLE
added homework 6

### PR DESCRIPTION
**Pull request by vithaiduchyk**
added homework_6 directory

### CODE REVIEW

File: Homework_6/divided_numbers.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+numbers = [1, 41, 156, 12, 51, 52, 5, 62, 65, 3, 78, 21, 134]
+divide_on = int(input("Enter number: "))
+divided_numbers = []
+
+for number in numbers:
+    if number % divide_on == 0:
+        divided_numbers.append(number)
+
+print(divided_numbers)
```
Comment (MarinaZelenska) [inline Homework_6/divided_numbers.py:9]: need to use formatting string

File: Homework_6/non_consecutive_number.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+numbers = [1, 5, 68, 0]
+prevNum = 0
+for number in range(1, len(numbers)):
+    if numbers[number] != numbers[number - 1] + 1:
+        print(f"The first non-consecutive number in the list: {numbers[number]}")
+        break
```
Comment (MarinaZelenska) [inline Homework_6/non_consecutive_number.py:6]: Missing block else for a loop. What if list is consecutive

File: Homework_6/divided_numbers.py (hunk 1 above)
Comment (vithaiduchyk) [reply Homework_6/divided_numbers.py:9]: Ok

File: Homework_6/non_consecutive_number.py (hunk 2 above)
Comment (vithaiduchyk) [reply Homework_6/non_consecutive_number.py:6]: Will fix

**Pull request merged**